src/WebJobs.Extensions.DocumentDB/DocumentDBAttribute.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// Attribute used to binds a parameter to an DocumentDB type.
+    /// </summary>
+    /// <remarks>
+    /// The method parameter type can be one of the following:
+    /// <list type="bullet">
+    /// <item><description><see cref="ICollector{T}"/></description></item>
+    /// <item><description><see cref="IAsyncCollector{T}"/></description></item>
```

Comment:
I just tested it. DocumentClient inherently supports JObject as well. So nothing fancy has to happen for node scripts to work.

---
In reply to: [56380664](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/47#discussion_r56380664) [](ancestors = 56380664,56370824)